lib/googleanalytics.js

```diff
@@ -21,15 +24,70 @@ module.exports = {
     ga('create', process.env.GA_ACCOUNT, 'auto');
     ga('send', 'pageview');
   },
+
   /**
    * pageview:
    * a proxy to basic GA pageview tracking
-   * @param  {[type]} path [description]
-   * @return {[type]}      [description]
+   * @param  {String} path - the current page page e.g. '/about'
    */
   pageview: function (path) {
     if (typeof ga === 'function') {
       ga('send', 'pageview', path);
     }
+  },
+
+  /**
+   * modalview:
+   * a proxy to basic GA pageview tracking
+   * @param  {String} modalName e.g. 'add-or-edit-club'
+   */
+  modalview: function (modalName) {
+    path = '/modal/' + modalName;
```

Comment:
Oh, aside from that, it would theoretically allow us to support people actually using those modals on browsers with JS disabled. Additional work beyond just making the modals have their own URLs would need to be done, though. Again though, not sure whether the modals are considered "progressive" or "required" functionality--I'd think at least the "learn more" would be required, but I suppose we could always have that link be a `mailto:` link w/ the progressive enhancement being the modal...

Anyways, that's off-topic for now I guess!